detector.go

```diff
@@ -22,6 +24,9 @@ const (
 
 var errFailedDetection = errors.New("no buildpacks participating")
 var errBuildpack = errors.New("buildpack(s) failed with err")
+var errInconsistentVersion = errors.New("top level version does not match metadata version")
+var errDoublySpecifiedVersions = errors.New("top level version cannot be specified along with metadata version; use metadata version instead")
```

Comment:
Should we also mention the buildpack name when these errors are printed, and update the language to look more like `warnTopLevelVersion`? @ekcasey

Reply:
We assumed that this would be needed - latest code has this change. We needed to set the condition code along with the error, since the error message will now vary according to the buildpack name.